Dockerfile

```diff
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1
+
+FROM openjdk:21-buster
```

Comment:
Shouldn't be version 19, to be consistent with other things.

Reply:
Set base image to openjdk:19-buster from openjdk:21-buster to make sure it is consistent with rest of the changes. Tested locally.